lib/tic_tac_toe.ex

```diff
@@ -0,0 +1,49 @@
+defmodule TicTacToe do
+
+  def run() do
+    start(new_game())
+  end
+
+  defp new_game() do
+    UI.introduce_ttt()
+    %Game{
+      board: %Board{size: 3, cells: Board.create_cells(3), marks: ["X", "O"]},
```

Comment:
As mentioned a couple of times, I would do something like `Board.empty(3)`.
And maybe come up with a good choice of words to add the `marks: ["X", "O"]`